src/Altinn.Platform/Altinn.Platform.Events/Events/Startup.cs

```diff
@@ -114,6 +118,27 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
             _logger.LogInformation("Startup // Configure");
 
+            ConsoleTraceService traceService = new ConsoleTraceService { IsDebugEnabled = true };
+
+            string workspacePath = Path.Combine(Environment.CurrentDirectory, "Migration");
+
+            if (Environment.CurrentDirectory.Contains("Test"))
+            {
+                workspacePath = Path.Combine(Environment.CurrentDirectory, @"..\..\..\Migration");
+            }     
+
+            app.UseYuniql(
+                new PostgreSqlDataService(traceService),
+                new PostgreSqlBulkImportService(traceService),
+                traceService,
+                new Yuniql.AspNetCore.Configuration
+                {
+                    WorkspacePath = workspacePath,
+                    ConnectionString = Configuration.GetValue<string>("PostgreSQLSettings:ConnectionString"),
```

Comment:
I get a feeling that unit tests now depends on having a running PostgreSQL server.

Reply:
you're right. Added parameter in appsettings to control whether or not the db is configured or not